addons/html_editor/static/src/core/delete_plugin.js

```diff
@@ -257,6 +257,7 @@ export class DeletePlugin extends Plugin {
             throw new Error("Invalid direction");
         }
         this.dispatchTo("delete_handlers");
+        this.dependencies.history.setStepExtra("canBeMerged", true);
```

Comment:
I suggest renaming `canBeMerged` to `batchable` because with "merge" we're kind of wondering "merging with what exactly"? The only reason I know the answer is because I know what the task is about... ^^' Whereas the concept of "batching" kind of implies it's for batching with other contiguous instances of the same object.